setup.py

```diff
@@ -78,6 +78,8 @@ def get_dir(system_path=None, virtual_path=None):
             'moduleframework-cmd = moduleframework.bashhelper:main',
             'modulelint = moduleframework.modulelint:main',
             'mtf-generator = moduleframework.mtf_generator:main',
+            'mtf-env-set = moduleframework.mtf_environment:mtfenvset',
```

Comment:
Could it be possle to use instead of avocado run *.py something like just mtf which will do this?